src/libpmem2/map.h

```diff
@@ -45,9 +45,7 @@ struct pmem2_map {
 	struct pmem2_vm_reservation *reserv;
 };
 
-#ifdef _WIN32
-os_rwlock_t split_merge_lock;
-#endif
+extern os_rwlock_t vm_rsv_lock;
```

Comment:
You should almost never use extern locks. Any extern variables need to have a *very* good reason for existing. This is not one.

Reply:
I've made *vm_rsv_lock* an extern so it would be initialized in map.c and shared between map_posix.c and map_windows.c files.